assess/shared/filters.py

```diff
@@ -34,12 +34,15 @@ def utc_to_bst(value, tz="Europe/London", export_format=False):
     else:
         raise ValueError("Invalid datetime format")
 
-    uk_time = utc_time.astimezone(timezone(tz))
+    if utc_time.tzinfo is None:
+        utc_time = utc.localize(utc_time)
```

Comment:
Small thing, but instead of defaulting to `utc`, default to `tz`'s timezone. Which it's likely still UTC, but at least it takes into account if tz is provided.

```python
default_timezone = timezone(tz)
utc_time = default_timezone.localize(utc_time)
```